### PR TITLE
Fix padding/fix_padding on GraphDirector

### LINKED ISSUE #109: GraphDirectory padding trait inconsistent
## Description

`GraphDirector.padding` exists, but the JS model uses `fit_padding`

## Reproduce

Try to do something with `GraphDirector.padding` and/or `GraphDirector.fit_padding`. Don't see padding.

## Expected behavior

See padding.

## Context

- Operating System and version: linux
- Browser and version: chromium latest
- JupyterLab version: 3.6.4
- IPyForceGraph version(s): 0.3.5

<details><summary>Required: installed server extensions</summary>
<pre>
  Paste the output from running `jupyter server extension list` (JupyterLab >= 3)
  or `jupyter serverextension list` (JupyterLab < 3) from the command line here.
  You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Required: installed lab extensions</summary>
<pre>
  Paste the output from running `jupyter labextension list` from the command line here.
  You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Troubleshoot Output</summary>
<pre>
  Paste the output from running `jupyter troubleshoot` from the command line here.
  You may want to sanitize the paths in the output.
</pre>
</details>

<details><summary>Command Line Output</summary>
<pre>
  Paste the output from your command line running `jupyter lab` here, use `--debug` if possible.
</pre>
</details>

<details><summary>Browser Output (recommended for all interface issues)</summary>
<pre>
  Paste the output from your browser JavaScript console replacing the text in here.

To learn how to open the developer tools in your browser:
https://developer.mozilla.org/en-US/docs/Learn/Common_questions/What_are_browser_developer_tools#How_to_open_the_devtools_in_your_browser
If too many messages accumulated after many hours of working in JupyterLab, consider
refreshing the window and then reproducing the bug to reduce the noise in the logs.

</pre>
</details>

### PR DESCRIPTION
**Pull request by nrbgt**
## Checklist

- [x] the PR target is correct
  - `main` for releases, docs-only fixes and post-release hot-fixes
  - [x] `dev` for everything else
- [x] ran `doit lint` locally

## References

- fixes #109
- includes #108

## Code changes

- [x] fix the names of traits on GraphDirector
- [x] update example
- [x] add regression test 
- [x] actually fail coverage on missed targets

## User-facing changes

- users will be able to update padding correctly

## Backwards-incompatible changes

- anything that used `padding` was broken

### CODE REVIEW

File: js/widgets/behaviors/graph-camera.ts [hunk 1]
```diff
@@ -89,7 +89,11 @@ export class GraphCameraModel extends ZoomBase implements IBehave {
       this.visible = is3d ? this.getVisible3d(zoom) : this.getVisible2d(zoom);
     }
 
-    this.save();
+    try {
+      this.save();
+    } catch {
+      // nothing to do here
```
Comment (sanbales) [inline js/widgets/behaviors/graph-camera.ts:95]: Should we throw a `console.log` here, or not worth to pollute that any further?

**Review by sanbales — APPROVED** [review]
Just a small stupid comment, code looks good.  I can try to test it but it may be better to have Joel test it for functionality.

File: js/widgets/behaviors/graph-camera.ts (hunk 1 above)
Comment (sanbales) [reply js/widgets/behaviors/graph-camera.ts:95]: Maybe we should invite [@JoelStansbury](https://github.com/JoelStansbury) to join `jupyrdf`.

Comment (nrbgt) [reply js/widgets/behaviors/graph-camera.ts:95]: right; it was throwing lots of errors when e.g. widgets get detached

**Pull request merged**